README.md

```diff
@@ -0,0 +1,31 @@
+# Octavia Provider Driver for F5 BigIP LTM devices
+
+This is the [Octavia](https://github.com/sapcc/octavia) provider driver for F5 BigIP LTM appliances.
+It communicates with BigIP devices via the declarative [AS3 API](https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/).
+
+Note: What we call 'worker' is called 'provider agent' in the [Octavia Provider Driver Development Guide](https://docs.openstack.org/octavia/latest/contributor/guides/providers.html).
```

Comment:
Would say that the worker is our own abstraction of an `agent`, since octavia doesn't support the agent concept out of the box. We don't really use the `Provider Agents` concept since it doesn't provide us with enough information for building a declarative specification.

Although, we use the driver-agent api, our implementation integrates more similiar to the upstream amphora driver and hooks itself into the database.